### PR TITLE
Updates signal dashboard to use new project personnel schema

### PR DESCRIPTION
**Pull request by johnclary**
## Associated issues
https://github.com/cityofaustin/atd-data-tech/issues/10766

## Testing
**URL to test:**  [Deploy preview](https://deploy-preview-874--atd-moped-main.netlify.app/moped/session/signin)

**Steps to test:**
1. Navigate to the [signal dashboard](https://deploy-preview-874--atd-moped-main.netlify.app/moped/views/signal-projects).
2. Observe that the page loads and there are users in the "Designer" and "Inspector columns. 
3. Add two team members to a project - set one as a designer and one as an inspector
4. Find your project in the signal dashboard table. Verify the "Designer" and "Inspector" columns populate correctly.

---
#### Ship list
- [ ] Code reviewed 
- [ ] Product manager approved
- [ ] Added to [QA test script](https://docs.google.com/spreadsheets/d/1n_O6MLh9cwwPf57HUM394Ea-z9uuoEV1-QW4axNZXLE/edit#) if applicable

### CODE REVIEW

File: moped-editor/src/queries/signals.js [hunk 1]
```diff
@@ -66,7 +65,9 @@ export const SIGNAL_PROJECTS_QUERY = gql`
         }
       }
       moped_proj_personnel(where: { is_deleted: { _eq: false } }) {
-        role_id
+        moped_proj_personnel_roles {
+          project_role_id
+        }
```
Comment (johnclary) [inline moped-editor/src/queries/signals.js:70]: everything else in this file is prettification

File: moped-toolbox/backfill_project_lead/run.py [hunk 2]
```diff
@@ -0,0 +1,171 @@
+"""Updates the `project_lead_id` column of moped_project records based on the the user who created the project.
```
Comment (chiaberry) [inline moped-toolbox/backfill_project_lead/run.py:1]: no harm pushing this to prod now, just wondering if you meant to include this along for the ride

**Review by chiaberry — APPROVED** [review]

File: moped-toolbox/backfill_project_lead/run.py (hunk 2 above)
Comment (johnclary) [reply moped-toolbox/backfill_project_lead/run.py:1]: ah crap i branched from main

**Review by frankhereford — APPROVED** [review]
Thanks John!

**Pull request merged**